plugins/org.jkiss.dbeaver.ext.oceanbase/src/org/jkiss/dbeaver/ext/oceanbase/ui/views/OceanbaseConnectionPage.java

```diff
@@ -0,0 +1,202 @@
+
+            hostText = new Text(hostGroup, SWT.BORDER);
+            gd = new GridData(GridData.FILL_HORIZONTAL);
+            gd.grabExcessHorizontalSpace = true;
+            hostText.setLayoutData(gd);
+            hostText.addModifyListener(textListener);
+
+            Label portLabel = UIUtils.createControlLabel(hostGroup,
+                    OceanbaseMessages.oceanbase_connection_page_label_port);
+            portLabel.setLayoutData(new GridData(GridData.HORIZONTAL_ALIGN_END));
+
+            portText = new Text(hostGroup, SWT.BORDER);
+            gd = new GridData(GridData.FILL_HORIZONTAL);
+            gd.grabExcessHorizontalSpace = true;
+            portText.setLayoutData(gd);
+            portText.addVerifyListener(UIUtils.getIntegerVerifyListener(Locale.getDefault()));
+            portText.addModifyListener(textListener);
+
+            Label databaseLabel = UIUtils.createControlLabel(hostGroup,
+                    OceanbaseMessages.oceanbase_connection_page_label_database);
+            databaseLabel.setLayoutData(new GridData(GridData.HORIZONTAL_ALIGN_END));
+
+            databaseText = new Text(hostGroup, SWT.BORDER);
+            gd = new GridData(GridData.FILL_HORIZONTAL);
+            gd.grabExcessHorizontalSpace = true;
+            databaseText.setLayoutData(gd);
+            databaseText.addModifyListener(textListener);
+
+            Label tenantLabel = UIUtils.createControlLabel(hostGroup,
+                    OceanbaseMessages.oceanbase_connection_page_label_tenant);
+            tenantLabel.setLayoutData(new GridData(GridData.HORIZONTAL_ALIGN_END));
+
+            tenantText = new Text(hostGroup, SWT.BORDER);
+            gd = new GridData(GridData.FILL_HORIZONTAL);
+            gd.grabExcessHorizontalSpace = true;
+            tenantText.setLayoutData(gd);
+            tenantText.addModifyListener(textListener);
+        }
+
+        createAuthPanel(addrGroup, 1);
+        createDriverPanel(addrGroup);
+        setControl(addrGroup);
+    }
+
+    @Override
+    public boolean isComplete() {
+        return super.isComplete() && hostText != null && databaseText != null && portText != null && tenantText != null
+                && !CommonUtils.isEmpty(hostText.getText()) && !CommonUtils.isEmpty(databaseText.getText())
+                && !CommonUtils.isEmpty(portText.getText()) && !CommonUtils.isEmpty(tenantText.getText());
+    }
+
+    @Override
+    public void loadSettings() {
+        // Load values from new connection info
+        DBPDataSourceContainer activeDataSource = site.getActiveDataSource();
+        DBPConnectionConfiguration connectionInfo = activeDataSource.getConnectionConfiguration();
+        if (urlText != null) {
+            if (CommonUtils.isEmpty(connectionInfo.getUrl())) {
+                try {
+                    saveSettings(site.getActiveDataSource());
+                } catch (Exception e) {
+                    setMessage(e.getMessage());

```

Comment:
Please log this exception. It can created like this:
```Java
private static final Log log = Log.getLog(OceanbaseConnectionPage.class);
```
Place this statement to the very top of the class, even above every other `static final` variables.